css/tutorial.styl

```diff
@@ -3,6 +3,10 @@
   max-width: 400px
   padding: 0 0 1.5em
 
+  @media screen and (max-width: 400px)
+    margin-left: 0vh
```

Comment:
`vh` is viewport height, so it doesn't make sense to use `vh` here. To be consistent with our other styles, `em` would be best. From MDN:

> The CSS properties line-height, font-size, margin-bottom and margin-top often has a value expressed in em.